karate-core/src/main/java/com/intuit/karate/ScriptValueMap.java

```diff
@@ -10,17 +10,18 @@
 public class ScriptValueMap extends HashMap<String, ScriptValue> {
 
     public static final String VAR_RESPONSE = "response";
-    public static final String VAR_RESPONSE_COOKIES = "responseCookies";
-    public static final String VAR_RESPONSE_HEADERS = "responseHeaders";
-    public static final String VAR_RESPONSE_STATUS = "responseStatus";
-    public static final String VAR_RESPONSE_TIME = "responseTime";
+    public static final String VAR_RESPONSE_COOKIES = VAR_RESPONSE+"Cookies";
```

Comment:
I know this is debatable, but I'd like to keep this file the way it is

Reply:
Do you mean no new constants in this file or you are not happy with concat?

Reply:
not happy with concat. just leave it as it is.

Reply:
done